glue_jupyter/widgets/axes_limits.py

```diff
@@ -0,0 +1,38 @@
+import traitlets
+import ipyvuetify as v
+from echo import delay_callback
+from ..link import dlink
+from ..state_traitlets_helpers import GlueState
+
+__all__ = ['AxesLimits']
+
+
+class AxesLimits(v.VuetifyTemplate):
+    template_file = (__file__, 'axes_limits.vue')
+
+    glue_state = GlueState().tag(sync=True)
+
+    x_min = traitlets.Float(default_value=None, allow_none=True).tag(sync=True)
+    x_max = traitlets.Float(default_value=None, allow_none=True).tag(sync=True)
+    y_min = traitlets.Float(default_value=None, allow_none=True).tag(sync=True)
+    y_max = traitlets.Float(default_value=None, allow_none=True).tag(sync=True)
+
+    def __init__(self, viewer_state):
+        super().__init__()
+        self.glue_state = viewer_state
+        dlink((self.glue_state, 'x_min'), (self, 'x_min'))
+        dlink((self.glue_state, 'x_max'), (self, 'x_max'))
+        dlink((self.glue_state, 'y_min'), (self, 'y_min'))
+        dlink((self.glue_state, 'y_max'), (self, 'y_max'))
+
+    def vue_apply_limits(self, data):
+        self._cache = (self.x_min, self.x_max, self.y_min, self.y_max)
+        with delay_callback(self.glue_state, 'x_min', 'x_max', 'y_min', 'y_max'):
+            if self.x_min is not None:
+                self.glue_state.x_min = self.x_min
+            if self.x_max is not None:
+                self.glue_state.x_max = self.x_max
+            if self.y_min is not None:
+                self.glue_state.y_min = self.y_min
+            if self.y_max is not None:
+                self.glue_state.y_max = self.y_max
```

Comment:
when used in an image viewer (now enabled), how is a change in _all_ limits, with equal aspect enabled, handled with this delayed callback?  From testing, it seems that the user-set x-limits are preserved and the y-limits are somehow adjusted to enforce the equal aspect?

I'm honestly not sure what the expected behavior should be in this case... maybe the smallest bounding box in x and y that contains the user-provided values?